alicloud/table_alicloud_vpc_route_entry.go

```diff
@@ -0,0 +1,253 @@
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/vpc"
+)
+
+//// TABLE DEFINITION
+
+func tableAlicloudVpcRouteEntry(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "alicloud_vpc_route_entry",
+		Description: "Alicloud VPC Route Entry",
+		List: &plugin.ListConfig{
+			ParentHydrate: listVpcRouteTable,
+			Hydrate:       listVpcRouteEntries,
+		},
+		// Get: &plugin.GetConfig{
+		// 	KeyColumns: plugin.AllColumns([]string{"route_table_id", "name"}),
+		// 	Hydrate:    getVpcRouteEntry,
+		// },
+		GetMatrixItem: BuildRegionList,
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the route entry.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("RouteEntryName"),
+			},
+			{
+				Name:        "route_entry_id",
+				Description: "The ID of the route entry.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "route_table_id",
+				Description: "The ID of the route table.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "status",
+				Description: "The status of the route entry.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "type",
+				Description: "The type of the route entry.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "description",
+				Description: "The description of the VRouter.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "instance_id",
+				Description: "The ID of the instance associated with the next hop.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "private_ip_address",
+				Description: "Specifies the private ip address for the route entry.",
+				Type:        proto.ColumnType_IPADDR,
+			},
+			{
+				Name:        "next_hop_oppsite_instance_id",
+				Description: "The ID of the instance associated with the next hop.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "next_hop_type",
+				Description: "The type of the next hop.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "ip_version",
+				Description: "The version of the IP protocol.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "destination_cidr_block",
+				Description: "The destination Classless Inter-Domain Routing (CIDR) block of the route entry.",
+				Type:        proto.ColumnType_CIDR,
+			},
+			{
+				Name:        "next_hop_region_id",
+				Description: "The region where the next hop instance is deployed.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "next_hop_oppsite_type",
+				Description: "The type of the next hop.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "next_hop_oppsite_region_id",
+				Description: "The region where the next hop instance is deployed.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "next_hops",
```

Comment:
@bigdatasourav What other data is contained in the `NextHops` response data? Can you please paste an example of the response data here?

Reply:
```
select jsonb_pretty(next_hops) as next_hops from alicloud_vpc_route_entry where route_entry_id = 'rte-0xigmoxq39x1uxx6y26d1' 
+---------------------------------------------------+
| next_hops                                         |
+---------------------------------------------------+
| [                                                 |
|     {                                             |
|         "Weight": 0,                              |
|         "Enabled": 0,                             |
|         "NextHopId": "vpn-0xinnicyu2izu6u6g0c38", |
|         "NextHopType": "VpnGateway",              |
|         "NextHopRegionId": "",                    |
|         "NextHopOppsiteType": "",                 |
|         "NextHopRelatedInfo": {                   |
|             "RegionId": "",                       |
|             "InstanceId": "",                     |
|             "InstanceType": ""                    |
|         },                                        |
|         "NextHopOppsiteRegionId": "",             |
|         "NextHopOppsiteInstanceId": ""            |
|     }                                             |
| ]                                                 |
+---------------------------------------------------+

Reply:
@bigdatasourav Sorry, I wasn't asking for the query result, I was asking for the response from Alicloud. What's in the `NextHops` field from Alicloud's response directly?

Reply:
It is  []NextHop